commerce_option.module

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * @file
+ * Defines the product option entity and associated features.
+ */
+
+use Drupal\Core\Render\Element;
+
+/**
+ * Implements hook_theme().
+ */
+function commerce_option_theme() {
+  return [
+    'commerce_option_option_value' => [
+      'render element' => 'elements',
+    ],
+  ];
+}
+
+/**
+ * Prepares variables for product option value templates.
+ *
+ * Default template: commerce-option-option-value.html.twig.
+ *
+ * @param array $variables
+ *   An associative array containing:
+ *   - elements: An associative array containing rendered fields.
+ *   - attributes: HTML attributes for the containing element.
```

Comment:
I believe these should be changed to option as well right?

Reply:
In this case it is just the passed in html attribute

Reply:
Ah, yeah, oops.